Bukkit/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ dependencies {
     implementation("org.bstats:bstats-bukkit")
 
     // Paper
-    compileOnly("io.papermc.paper:paper-api")
+    compileOnly("dev.folia:folia-api:1.19.4-R0.1-SNAPSHOT")
```

Comment:
```suggestion
    compileOnly("dev.folia:folia-api:1.19.4-R0.1-SNAPSHOT")
```
This doesn't look right to me. The bukkit module has to stick to Paper for compatibility and compiling reasons.

Reply:
When i allow both. The the API support for folia will break in IJ

Reply:
But this breaks compatibility with Spigot and Paper.

Reply:
Okay. Let me check that how we can handle it